src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -3421,6 +3421,165 @@ adjset_uniform(Node &res)
     }
 }
 
+//-----------------------------------------------------------------------------
+void 
+polyhedral_chain(index_t howmany, // how long the chain ought to be
+                 Node &chain)
+{
+    Node &chain_coords = chain["coordsets/coords"];
+    Node &chain_topo = chain["topologies/topo"];
+    Node &chain_fields = chain["fields"];
+
+    chain_coords["type"] = "explicit";
+
+    // Each cube requires 8 vertices, and each triangular prism requires 6
+    // There are two prisms for every cube, so 8+6+6 = 20 vertices
+    chain_coords["values/x"].set(conduit::DataType::int64(howmany * 20));
+    chain_coords["values/y"].set(conduit::DataType::int64(howmany * 20));
+    chain_coords["values/z"].set(conduit::DataType::int64(howmany * 20));
+
+    int64* values_x = chain_coords["values/x"].value();
+    int64* values_y = chain_coords["values/y"].value();
+    int64* values_z = chain_coords["values/z"].value();
```

Comment:
Very minor point, but Conduit style is generally to left-associate pointers and references, e.g.:

```
    int64 *values_x = chain_coords["values/x"].value();
    int64 *values_y = chain_coords["values/y"].value();
    int64 *values_z = chain_coords["values/z"].value();
```